pac/tasks/upload-sbom-to-trustification.yaml

```diff
@@ -262,7 +262,7 @@ spec:
           -H "transfer-encoding: chunked" \
           -H "content-type: application/json" \
           --data "@$supported_version_of_sbom" \
-          "$bombastic_api_url/api/v2/sbom?id=$sbom_id"
+          "$bombastic_api_url/api/v1/sbom?id=$sbom_id"
```

Comment:
This is not intended. We need to update the task in build-definitions so this doesn't happen.